solr/solr-ref-guide/modules/query-guide/pages/learning-to-rank.adoc

```diff
@@ -113,7 +113,7 @@ In the form of JSON files your trained model or models (e.g., different models f
 
 == Quick Start with LTR
 
-The `"techproducts"` example included with Solr is pre-configured with the plugins required for learning-to-rank, but they are disabled by default.
+The `"techproducts"` example included with Solr is pre-configured with the `ltr` module and plugins required for learning-to-rank, but they are disabled by default.
```

Comment:
To be precise, there is no way to load a **module** from solrconfig.xml, but you can load all the jars in a module's `lib/` folder, thus enabling all that modules' plugins.
```suggestion
The `"techproducts"` example included with Solr is pre-configured to load the plugins required for learning-to-rank from the `ltr` module, but they are disabled by default.
```

Reply:
added a note about this to `solr-modules.adoc`